### ISSUE TITLE
ejabberd_stun TCP port 443 binds to all interfaces

### ISSUE THREAD

**Issue by pwarmen3** [processone/ejabberd#529]
I have tried listening on only TCP port 443 as well to the same effect. How can I get ejabberd to bind to only x.x.x.x?

ejabberd.yml

```
  -
    port: 443
    use_turn: true
    turn_ip: "x.x.x.x"
    ip: "x.x.x.x"
    transport: tcp
    module: ejabberd_stun
  -
    port: 443
    use_turn: true
    turn_ip: "x.x.x.x"
    ip: "x.x.x.x"
    transport: udp
    module: ejabberd_stun
```
ejabberdctl live tells me its listening on the designated interfaces but it really is not

```
 [debug] Supervisor ejabberd_listeners started ejabberd_listener:start({443,{x,x,x,x},tcp}, ejabberd_stun, [{turn_ip,<<"x.x.x.x">>},{use_turn,true}]) at pid <0.460.0>
 [info] ejabberd 15.03.90 is started in the node ejabberd@localhost
 [debug] Supervisor ejabberd_listeners started ejabberd_listener:start({443,{x,x,x,x},udp}, ejabberd_stun, [{turn_ip,<<"x.x.x.x">>},{use_turn,true}]) at pid <0.461.0>
```
netstat -nap

```
tcp        0      0 0.0.0.0:443             0.0.0.0:*               LISTEN      19929/beam.smp
udp        0      0 x.x.x.x:443       0.0.0.0:*                           19929/beam.smp
```

**Comment by haegar** [comment]
Please try current git, most likely the same problem and fix as #530

**Comment by pwarmen3** [comment]
yes, the new git fixed it
